docs/raspberry-pi5.md

```diff
@@ -0,0 +1,331 @@
+        CONFIG_NF_CONNTRACK_IPV4: IPv4 connection tracking support (required for NAT): unknown (warning)
+        CONFIG_NF_REJECT_IPV4: IPv4 packet rejection: module (pass)
+        CONFIG_NF_NAT_IPV4: IPv4 NAT: unknown (warning)
+        CONFIG_IP_NF_IPTABLES: IP tables support: module (pass)
+          CONFIG_IP_NF_FILTER: Packet filtering: module (pass)
+            CONFIG_IP_NF_TARGET_REJECT: REJECT target support: module (pass)
+          CONFIG_IP_NF_NAT: iptables NAT support: module (pass)
+          CONFIG_IP_NF_MANGLE: Packet mangling: module (pass)
+        CONFIG_NF_DEFRAG_IPV4: module (pass)
+        CONFIG_NF_CONNTRACK_IPV6: IPv6 connection tracking support (required for NAT): unknown (warning)
+        CONFIG_NF_NAT_IPV6: IPv6 NAT: unknown (warning)
+        CONFIG_IP6_NF_IPTABLES: IP6 tables support: module (pass)
+          CONFIG_IP6_NF_FILTER: Packet filtering: module (pass)
+          CONFIG_IP6_NF_MANGLE: Packet mangling: module (pass)
+          CONFIG_IP6_NF_NAT: ip6tables NAT support: module (pass)
+        CONFIG_NF_DEFRAG_IPV6: module (pass)
+      CONFIG_BRIDGE: 802.1d Ethernet Bridging: module (pass)
+        CONFIG_LLC: module (pass)
+        CONFIG_STP: module (pass)
+    CONFIG_EXT4_FS: The Extended 4 (ext4) filesystem: built-in (pass)
+    CONFIG_PROC_FS: /proc file system support: built-in (pass)
+  ```
+
+### Deploy a Node using k0s
+
+If you want a more hands-on process for setting up your Pi 5 devices as Kubernetes nodes with the k0s binary, refer to [the guide for Pi devices](./raspberry-pi4.md).
+
+## Deploy a Node using k0sctl
+
+### Install k0sctl on Your Host
+
+Follow [the k0sctl installation guide](https://github.com/k0sproject/k0sctl#installation) and install k0sctl on your host machine.
+
+### Single node K0s cluster
+
+For this example, we'll create a cluster.yaml that describes your known Pi 5 device and use it as a single node (controller & worker) cluster, for example:
+
+  ```yaml
+  apiVersion: k0sctl.k0sproject.io/v1beta1
+  kind: Cluster
+  metadata:
+    name: k0s-cluster
+    user: admin
+  spec:
+    hosts:
+    - ssh:
+        address: <IP_ADDRESS_OF_THE_SERVER>
+        user: <YOUR_USER_NAME>
+        port: 22
+        keyPath: ~/.ssh/id_rsa
+      role: controller+worker
+  ```
+
+### SSH agent
+
+  By default, k0sctl doesn’t prompt you for passphrases, so the easiest solution is to load your key into an SSH agent before running k0sctl. Here’s how you can do it:
+
+  1. Start the SSH agent (if not already running)
+
+      ```bash
+      eval "$(ssh-agent -s)"
+      ```
+
+  2. ​​Add your private key (you’ll be prompted for the passphrase)
+
+      ```bash
+      ssh-add ~/.ssh/id_rsa
+      ```
+
+  3. Verify the key is loaded
+
+      ```bash
+      ssh-add -l
+      ```
+
+### Deploy cluster
+
+  1. Apply the cluster.yaml using k0sctl on your local machine.
+
+      ```bash
+      ➜ k0sctl apply --config cluster.yaml
```

Comment:
Whenever a console command including output is shown, it's conventional to use "console", along with the dollar sign as a shell prompt (or a hashbang if the root user is assumed).

```suggestion
      ```console
      $ k0sctl apply --config cluster.yaml
```